app/services/world_location_base_path.rb

```diff
@@ -0,0 +1,20 @@
+class WorldLocationBasePath
+  EXCEPTIONAL_CASES = {
```

Comment:
We most certainly can. Added in 	0b4e917. We appreciate this isn't ideal. Would very much welcome a chat with you next week to discuss options for a better solution.